<?xml version="1.0" encoding="UTF-8"?><testrun name="FactorialTest" project="DSA" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="com.turing.dsa.recursion.FactorialTest" time="0.008" displayname="FactorialTest" uniqueid="[engine:junit-jupiter]/[class:com.turing.dsa.recursion.FactorialTest]">
    <testcase name="test" classname="com.turing.dsa.recursion.FactorialTest" time="0.008" displayname="test()" uniqueid="[engine:junit-jupiter]/[class:com.turing.dsa.recursion.FactorialTest]/[method:test()]">
      <failure>
        <expected>6</expected>
        <actual>24</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;6&gt; but was: &lt;24&gt;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:531)
	at com.turing.dsa.recursion.FactorialTest.test(FactorialTest.java:19)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)


      </failure>
    </testcase>
  </testsuite>
</testrun>
